migrations/20220312215246-create-demands.js

```diff
@@ -0,0 +1,99 @@
+'use strict';
+
+module.exports = {
+  async up(queryInterface, Sequelize) {
+    await queryInterface.sequelize.transaction(async (transaction) => {
+      await queryInterface.createTable(
+        'demands',
+        {
+          id: {
+            type: Sequelize.UUID,
+            allowNull: false,
+            primaryKey: true,
+            unique: true,
+            autoIncrement: false,
+          },
+          comment: {
+            type: Sequelize.STRING,
+          },
+          place_id: {
+            type: Sequelize.UUID,
+            allowNull: false,
+          },
+          supply_id: {
+            type: Sequelize.UUID,
+            allowNull: false,
+          },
+          priority_id: {
+            type: Sequelize.UUID,
+          },
+          created_at: {
+            type: Sequelize.DATE,
+            defaultValue: Sequelize.NOW,
+            allowNull: false,
+          },
+          updated_at: {
+            type: Sequelize.DATE,
+            defaultValue: Sequelize.NOW,
+            allowNull: false,
+          },
+        },
+        { transaction },
+      );
+
+      await queryInterface.addConstraint('places', {
+        type: 'primary key',
+        name: 'places_pkey',
+        fields: ['id'],
+        transaction,
+      });
```

Comment:
I think you could achieve association by adding it in the `demands` table creation migration :
```
place_id: {
        type: Sequelize.UUID,
        references: {
          model: 'places', // name of Target model
          key: 'id', // key in Target model that we're referencing
        }
      }
```

Does that make sense? Or am I missing something?

Reply:
I can change the creation of FKs to `references` syntax but `place_id` won't work as it is not PK. We need to add the PK constraint to it first.

Reply:
So if you don't mind I can merge  my code so it will be a primary key :) I have it already done in migrations

Reply:
Here I have it as a primary key
https://github.com/copotrzebne-pl/copotrzebne-api/pull/6

Reply:
OK, merged.